client/reader/stream/style.scss

```diff
@@ -655,98 +652,93 @@
 		line-height: 18px;
 	}
 
-	.stream__right-column {
```

Comment:
At this point and below for this file, there are no css property changes.

Several blocks are being moved from `.stream__two-column .stream__right-column` to `.stream__right-column` so the styles apply to the right column on both two column and tabbed views.